styles.css

```diff
@@ -142,4 +146,64 @@
                 position: relative;
                 display: flex;
                 flex-direction: column;
-            }
\ No newline at end of file
+                align-items: center;
+            }
+
+        @media only screen and (max-width: 700px) {
+            .header{
+                width: auto;
+            }        
+              }
+        @media only screen and (max-width: 320px){
+            .header{
+                flex-direction: column;
+                padding-top: 3em;
+                padding-bottom: 3em;
+            }
+
+            .navbar{
+                flex-direction: column;
+                align-items: center;
+                max-width: 350px;
+                gap: 0;
+            }
+
+            .right-nav{
+                gap: 15px;
+            }
+
+            #MQ--buttons{
+                display: block;
```

Comment:
good for changing designs for buttons. 
But there was no need to create separate HTML and hide it under some conditions. You could overwrite the CSS of those elements in mediaquery.

Reply:
according to our discussion  we'll leave it as it is)